src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -63,6 +65,10 @@ public EditCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_ADDRESS).isPresent()) {
             editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()));
         }
+        if (argMultimap.getValue(PREFIX_APPOINTMENT).isPresent()) {
+            editPersonDescriptor.setAppointment(ParserUtil
```

Comment:
consider using intermediate variables to improve readability?

instead of like
`x.set(P.parse(arg.getVal(P_A).get())`

you can do something like
```java
rawAppointment = argMm.getVal(P_A).get();
newAppointment = ParserUtil.parseApp(rawAppointment);
editPersonDescriptor.setAppt(newAppointment);
```